build.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
```

Comment:
please address shellcheck  warnings https://www.shellcheck.net/

```
Line 12:
  case $c in
  ^-- SC2220: Invalid flags are not handled. Add a *) case.
 
Line 18:
if [[ -z ${VERSION} ]]; then
   ^-- SC2039: In POSIX sh, [[ ]] is undefined.
 
Line 23:
platforms=("linux/amd64" "darwin/amd64" "windows/amd64")
          ^-- SC2039: In POSIX sh, arrays are undefined.
 
Line 25:
for platform in "${platforms[@]}"
                 ^-- SC2039: In POSIX sh, array references are undefined.
 
Line 27:
    platform_split=(${platform//\// })
                   ^-- SC2039: In POSIX sh, arrays are undefined.
                    ^-- SC2039: In POSIX sh, string replacement is undefined.
                    ^-- SC2206: Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.
 
Line 28:
    GOOS=${platform_split[0]}
         ^-- SC2039: In POSIX sh, array references are undefined.
 
Line 29:
    GOARCH=${platform_split[1]}
           ^-- SC2039: In POSIX sh, array references are undefined.
 
Line 34:
    env GOOS=${GOOS} GOARCH=${GOARCH} go build -o ${OUTPUT_DIR}/${output_name}
             ^-- SC2086: Double quote to prevent globbing and word splitting.
                            ^-- SC2086: Double quote to prevent globbing and word splitting.
                                                  ^-- SC2086: Double quote to prevent globbing and word splitting.
>>                                                              ^-- SC2086: Double quote to prevent globbing and word splitting.
```